apps/api/src/config/env-validator.ts

```diff
@@ -91,7 +91,19 @@ if (process.env.STORAGE_SERVICE === 'AWS' || !process.env.STORAGE_SERVICE) {
   validators.AWS_SECRET_ACCESS_KEY = str();
 }
 
-if (process.env.NODE_ENV !== 'local' && process.env.NODE_ENV !== 'test') {
+if (process.env.NODE_ENV === 'regression') {
+  const noEmpty = makeValidator((dopplerToken) => {
+    if (!dopplerToken) {
+      throw new Error('Expected a non empty Doppler token');
+    }
+  });
+
+  validators.DOPPLER_TOKEN = noEmpty({
+    desc: 'Doppler read-only access token to get the secrets needed to run regression tests',
+  });
```

Comment:
This validator will help to remind us to add the DOPPLER_TOKEN value when in need to execute the regression tests. Should only be available in this environment purposedly created.